scripts/remove-default-apps.ps1

```diff
@@ -27,12 +27,12 @@ $apps = @(
     "Microsoft.People"
     "Microsoft.SkypeApp"
     #"Microsoft.Windows.Photos"
-    "Microsoft.WindowsAlarms"
+    #"Microsoft.WindowsAlarms"
```

Comment:
Windows Alarms is not a bloatware. There is not many efficient apps in the Windows store to replace such useful thing.